app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -706,6 +743,23 @@ public void setCamera(Camera camera) {
 		}
 	}
 
+
+	@Override
+	protected void onDraw(Canvas canvas)
+	{
+		if (GRID_ENABLED) {
+			DisplayMetrics metrics = Resources.getSystem().getDisplayMetrics();
+			int screenWidth = metrics.widthPixels;
```

Comment:
Use the functions already available in `Utils.java` for screen width and height.